opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -26,11 +26,19 @@
     ScopeManager scopeManager();
 
     /**
-     * @return the active {@link Span}. This is a shorthand for Tracer.scopeManager().active().span(),
-     * and null will be returned if {@link Scope#active()} is null.
+     * @return the active {@link Span}. This is a shorthand for {@code Tracer.scopeManager().activeSpan()}.
      */
     Span activeSpan();
 
+    /**
+     * Make a {@link Span} instance active for the current context (usually a thread).
+     * This is a shorthand for {@code Tracer.scopeManager().activate(span)}.
+     *
+     * @return a {@link Scope} instance to control the end of the active period for the {@link Span}. It is a
+     * programming error to neglect to call {@link Scope#close()} on the returned instance.
```

Comment:
and may lead to memory leaks and the scope may remain in the thread-local stack.